opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -178,15 +179,22 @@
             "metadata": {
                 "description": "HealthCheck cache interval value in seconds."
             }
+        },
+        "customTShirtSize": {

```

Comment:
maybe call it custom deployment size? We know waht we mean by t-shirt size, do others?

Reply:
Agree'd I'll fix up to `deploymentSize`